samples/sampler/stories/qa/VirtualGridList.js

```diff
@@ -8,6 +8,7 @@ import {VirtualGridList} from '@enact/sandstone/VirtualList';
 import {mergeComponentMetadata} from '@enact/storybook-utils';
 import {action} from '@enact/storybook-utils/addons/actions';
 import {boolean, number, select} from '@enact/storybook-utils/addons/controls';
+import ilib from '@enact/i18n';
```

Comment:
should be in alphabetical order after `from` so, it should be placed above line 1.

Reply:
This line is removed.